### PR TITLE
Fix for atlas compilation

### PR DESCRIPTION
**Pull request by abretaud**
This shold fix part of #815 issue

### CODE REVIEW

File: packages/package_atlas_3_10/tool_dependencies.xml [hunk 1]
```diff
@@ -48,9 +48,9 @@
                     <action type="shell_command"><![CDATA[
                         cd ATLAS &&
                         mkdir build &&
-                        patch -p1 </host/static_full_blas_lapack.diff &&
-                        patch -p1 </host/shared_libraries.diff &&
-                        patch -p1 </host/cpu-throttling-check.diff &&
+                        patch -p1 < ../static_full_blas_lapack.diff &&
+                        patch -p1 < ../shared_libraries.diff &&
+                        patch -p1 < ../cpu-throttling-check.diff &&
```
Comment (nsoranzo) [inline packages/package_atlas_3_10/tool_dependencies.xml]: I think also the file names are wrong, shouldn't they be the ones downloaded in lines 43-45?

**Comment by nsoranzo** [comment]
Not sure this fixes everything, but a good step forward, thanks @abretaud!

**Comment by abretaud** [comment]
Yep, I have no clue for the problem with precompiled binaries...

**Comment by nsoranzo** [comment]
Pushed to MTS.

**Pull request merged**